### PR TITLE
srpbs qc

### PR DESCRIPTION
**Pull request by clarkenj**
As suggested I tried looping through the srpbs subjects and running the qc per participant. It ran on all subjects that had data, and I can't see anything untoward in the .err or .out files, so I don't think it was a subject causing the error. 

Since I wanted to run qc for this cohort at the participant level anyway, can I use the results as my final qc?

### CODE REVIEW

File: srpbs/submit_qc_single.sh [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+    # Check if the directory contains .html files and skip it if it does
+        if [[ -n $(find "$subject_dir" -maxdepth 1 -type f -name "*.html") ]]; then
+        echo "Skipping subject directory with .html files: $subject_dir"
+        continue
+    fi
+
+    subject_label=$(basename $subject_dir)
+    subject_label=${subject_label#sub-}  # Remove "sub-"
+
+    # Check if the subject number is greater than or equal to "0468"
```
Comment (clarkenj) [inline srpbs/submit_qc_single.sh:26]: this is because I ran the script once and it timed out, so I ran the rest of the subjects separately

**Comment by htwangtw** [comment]
> I can't see anything untoward in the .err or .out files, so I don't think it was a subject causing the error.

I am not sure what you are trying to convey here: sounds like you did run into an error / undesired behaviour, but it's not clear to me what is it?

**Comment by clarkenj** [comment]
Sorry, I meant [this error](https://github.com/SIMEXP/giga_auto_qc/issues/19#issuecomment-1697622561) which we thought might be due to a subject, hence the looping

**Comment by htwangtw** [comment]
Got it! Does that mean you have some result .tsv files?

**Comment by clarkenj** [comment]
Yes one per participant which I have combined into one .tsv

**Review by htwangtw — APPROVED** [review]
Forgot to approve this one. since the data is archived so all is good!

**Pull request merged**